exchange/exchangetest/case-insensitive-alternate-bidder-codes.json

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
I think we can cut about 200 lines out of this file and achieve the same level of alternate bidder code coverage you've got here. IMO, something like this helps keep the test focused:
```
{
    "incomingRequest": {
        "ortbRequest": {
            "id": "some-request-id",
            "site": {
                "page": "test.somepage.com"
            },
            "imp": [
                {
                    "id": "imp-id-1",
                    "video": {
                        "mimes": [
                            "video/mp4"
                        ]
                    },
                    "ext": {
                        "prebid": {
                            "bidder": {
                                "pubmatic": {
                                    "publisherId": 5890
                                },
                                "appnexus": {
                                    "placementId": 1
                                }
                            }
                        }
                    }
                }
            ],
            "ext": {
                "prebid": {
                    "alternatebiddercodes": {
                        "enabled": true,
                        "bidders": {
                            "PUBmatic": {
                                "enabled": true,
                                "allowedbiddercodes": [
                                    "groupm"
                                ]
                            }
                        }
                    }
                }
            }
        }
    },
    "outgoingRequests": {
        "pubmatic": {
            "expectRequest": {
                "ortbRequest": {
                    "id": "some-request-id",
                    "site": {
                        "page": "test.somepage.com"
                    },
                    "imp": [
                        {
                            "id": "imp-id-1",
                            "video": {
                                "mimes": [
                                    "video/mp4"
                                ]
                            },
                            "ext": {
                                "bidder": {
                                    "publisherId": 5890
                                }
                            }
                        }
                    ],
                    "ext": {
                        "prebid": {
                            "alternatebiddercodes": {
                                "enabled": true,
                                "bidders": {
                                    "pubmatic": {
                                        "enabled": true,
                                        "allowedbiddercodes": [
                                            "groupm"
                                        ]
                                    }
                                }
                            }
                        }
                    }
                }
            },
            "mockResponse": {
                "pbsSeatBids": [
                    {
                        "pbsBids": [
                            {
                                "ortbBid": {
                                    "id": "pubmatic-bid-1",
                                    "impid": "imp-id-1",
                                    "price": 0.71
                                },
                                "bidType": "video",
                                "bidMeta": {
                                    "adaptercode": "pubmatic"
                                }
                            }
                        ],
                        "seat": "pubmatic"
                    },
                    {
                        "pbsBids": [
                            {
                                "ortbBid": {
                                    "id": "pubmatic-bid-2",
                                    "impid": "imp-id-1",
                                    "price": 0.51
                                },
                                "bidType": "video",
                                "bidMeta": {
                                    "adaptercode": "pubmatic"
                                }
                            }
                        ],
                        "seat": "groupm"
                    }
                ]
            }
        },
        "appnexus": {
            "expectRequest": {
                "ortbRequest": {
                    "id": "some-request-id",
                    "site": {
                        "page": "test.somepage.com"
                    },
                    "imp": [
                        {
                            "id": "imp-id-1",
                            "video": {
                                "mimes": [
                                    "video/mp4"
                                ]
                            },
                            "ext": {
                                "bidder": {
                                    "placementId": 1
                                }
                            }
                        }
                    ],
                    "ext": {
                        "prebid": {
                            "alternatebiddercodes": {
                                "enabled": true,
                                "bidders": null
                            }
                        }
                    }
                }
            },
            "mockResponse": {
                "pbsSeatBids": [
                    {
                        "pbsBids": [
                            {
                                "ortbBid": {
                                    "id": "appnexus-bid-1",
                                    "impid": "imp-id-1",
                                    "price": 0.3
                                },
                                "bidType": "banner",
                                "bidMeta": {
                                    "adaptercode": "appnexus"
                                }
                            }
                        ],
                        "seat": "appnexus"
                    },
                    {
                        "pbsBids": [
                            {
                                "ortbBid": {
                                    "id": "appnexus-bid-2",
                                    "impid": "imp-id-1",
                                    "price": 0.3
                                },
                                "bidType": "banner",
                                "bidMeta": {
                                    "adaptercode": "appnexus"
                                }
                            }
                        ],
                        "seat": "groupm"
                    }
                ]
            }
        }
    },
    "response": {
        "bids": {
            "id": "some-request-id",
            "seatbid": [
                {
                    "seat": "groupm",
                    "bid": [
                        {
                            "id": "pubmatic-bid-2",
                            "impid": "imp-id-1",
                            "price": 0.51,
                            "ext": {
                                "origbidcpm": 0.51,
                                "prebid": {
                                    "meta": {
                                        "adaptercode": "pubmatic"
                                    },
                                    "type": "video"
                                }
                            }
                        },
                        {
                            "id": "appnexus-bid-2",
                            "impid": "imp-id-1",
                            "price": 0.3,
                            "ext": {
                                "origbidcpm": 0.3,
                                "prebid": {
                                    "meta": {
                                        "adaptercode": "appnexus"
                                    },
                                    "type": "banner"
                                }
                            }
                        }
                    ]
                },
                {
                    "seat": "pubmatic",
                    "bid": [
                        {
                            "id": "pubmatic-bid-1",
                            "impid": "imp-id-1",
                            "price": 0.71,
                            "ext": {
                                "origbidcpm": 0.71,
                                "prebid": {
                                    "meta": {
                                        "adaptercode": "pubmatic"
                                    },
                                    "type": "video"
                                }
                            }
                        }
                    ]
                },
                {
                    "seat": "appnexus",
                    "bid": [
                        {
                            "id": "appnexus-bid-1",
                            "impid": "imp-id-1",
                            "price": 0.3,
                            "ext": {
                                "origbidcpm": 0.3,
                                "prebid": {
                                    "meta": {
                                        "adaptercode": "appnexus"
                                    },
                                    "type": "banner"
                                }
                            }
                        }
                    ]
                }
            ]
        }
    }
}

```